src/bot.js

```diff
@@ -91,7 +99,28 @@ class Bot {
         'skin_xcaf',
       );
 
-    this.websocket.emit('falcon:create',{ url: config.falconUrl, payload });
+    this.websocket.emit('falcon:create', { url: config.falconUrl, payload });
+
+    return {
+      skin,
+      link: this.getLink(componentName, slug),
+      payload
+    };
+  }
+
+  falconDestroy(pr, skinInfo) {
+    let skin = skinInfo.skinName;
+
+    // normalize cresus name
+    if (skin === 'cresuscasino') {
```

Comment:
Maybe move this to the `getFalconComponentName`, then you can do just 
```
const componentName = this.getFalconComponentName(skinInfo.skinName);
```

Reply:
good idea, fixed